presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGeneratorSql.java

```diff
@@ -211,4 +216,27 @@ public void testDistinctSelection()
                 defaultSessionHolder,
                 ImmutableMap.of());
     }
+
+    @Override
+    @Test
+    public void testDistinctLimitPushdown()
```

Comment:
How about testing something with a distinct limit on top of an existing group by ? What about on top of an existing global aggregation ?

Reply:
adding assertions on existing aggregations/groupbys.